tasks/server_keys.yml

```diff
@@ -93,7 +93,7 @@
     dest: "{{ openvpn_key_dir }}/ca.conf"
     owner: root
     group: root
-    mode: 744
+    mode: 0644
```

Comment:
This doesn't need to be executable. As you can see, it wasn't executable previously (having permissions `440`) and it worked.